CDCgov/cdc-open-viz
# Dev 8596

## Testing Steps

1. Create/load line chart
2. Select dynamic data series checkbox is series section of editor
3. Change line type setting and ensure that it reflects in preview
4. If line type is solid, ensure curve type dropdown appears and reflects in preview
5. Ensure these dropdowns dont show for other chart types

## Self Review

- I have added testing steps for reviewers
- I have commented my code, particularly in hard-to-understand areas
- My changes generate no new warnings
- New and existing unit tests are passing

## Screenshots (if applicable)



## Additional Notes
